apiserver/src/auth/authorizor.rs

```diff
@@ -0,0 +1,438 @@
+//! This module provides abstractions for authenticating and authorizing requests from brupop agents to make changes to
+//! their Node's resources (including BottlerocketNode custom resources, or Draining their host Nodes of Pods.)
```

Comment:
nit: so it doesn't read as much like Victorian era writing:
```suggestion
//! the underlying Node's resources (including BottlerocketNode custom resources, or draining the Node of Pods.)
```